adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/net45/WindowsFormsWebAuthenticationDialogBase.cs

```diff
@@ -251,6 +251,27 @@ private bool CheckForClosingUrl(Uri url)
             return canClose;
         }
 
+        private string GetAuthenticationCode(HtmlDocument document)
```

Comment:
Could you get this covered by unit tests?

Reply:
@bgavrilMS Added unit test